contracts/L1Teleporter.sol

```diff
@@ -234,4 +234,9 @@ contract L1Teleporter is Pausable, AccessControl, L2ForwarderPredictor, IL1Telep
         results.l2l3TokenBridgeCost =
             gasParams.l2l3TokenBridgeMaxSubmissionCost + gasParams.l2l3TokenBridgeGasLimit * gasParams.l3GasPriceBid;
     }
+
+    /// @dev Alias the address if it is a contract
```

Comment:
also note that it does not work in certain edge case e.g. within constructor

Reply:
added a few comments